### PR TITLE
SR2020 New competition date

### PR DESCRIPTION
**Pull request by antoinepetty**

### CODE REVIEW

File: _events/sr2020/competition.md [hunk 1]
```diff
@@ -13,7 +14,7 @@ Keep an eye on this page for updates as more information becomes available.
 
 ## Schedule
 
-The competition will be held on a Saturday and Sunday in late June / early July.
+The 2-day competition will be held on the weekend of Saturday 4th July.
```
Comment (PeterJCLaw) [inline _events/sr2020/competition.md]: nit: it's slightly nicer to spell out numbers when they're used as words:
```suggestion
The two-day competition will be held on the weekend of Saturday 4th July.
```

File: covid-19.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Last updated: {{ page.date | date: "%e %B %Y" }}.
 
 ## The Competition
 
-Following guidance from the UK government and the University of Reading, **we will need to postpone Student Robotics 2020** from the original date in April, until some time late June / early July.
+Following guidance from the UK government and the University of Reading, we are postponing Student Robotics 2020 from the original date in April, until the **weekend of the 4th July** (Subject to change).
```
Comment (PeterJCLaw) [inline covid-19.md]: grammar nit:
```suggestion
Following guidance from the UK government and the University of Reading, we are postponing Student Robotics 2020 from the original date in April, until the **weekend of the 4th July** (subject to change).
```

Comment (PeterJCLaw) [inline covid-19.md]: Perhaps expand a little on what might cause this to change?
```suggestion
Following guidance from the UK government and the University of Reading, we are postponing Student Robotics 2020 from the original date in April, until the **weekend of the 4th July** (subject to review mid-May).
```

File: _posts/2020-03-28-sr2020-new-provisional-date.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+layout: news
+title: Student Robotics 2020 New Provisional Competition Date
+---
+
+## Provisional Competition Date
+
+After surveying our SR2020 teams and volunteers, we have selected the **weekend of the 4th July 2020** to be the most appropriate date for the competition.
+
+The COVID-19 situation changes often, so we will re-evaluate in mid-May.
```
Comment (RealOrangeOne) [inline _posts/2020-03-28-sr2020-new-provisional-date.md]: I think we need to change the wording of this slightly. As I see it, it's either this weekend, or no weekend at all (or a heavy modification of a usual competition cycle).

**Pull request merged**